.github/workflows/build-and-publish-docker.yml

```diff
@@ -34,63 +114,127 @@ jobs:
       id-token: write
 
     steps:
+      - name: Set Swap Space
+        uses: pierotofy/set-swap-space@master
```

Comment:
Do we need master? Don't want to use a tag / version?

I'm happy to put the PR up for dependabot to make PRs and keep this up to date if we're interested too. Something like: https://github.com/psf/black/blob/main/.github/dependabot.yml#L3-L10

Reply:
It hasn't been updated in 2 years, but agreed. Can make the change since it has a v1.0 tag